src/pages/console/adminui/access-management.mdx

```diff
@@ -41,3 +37,44 @@ Follow these instructions to add and manage team members and their access to a p
 <Callout warning>
 If a team member logs into Amplify Studio, their login token is valid for 60 minutes, unless explicitly logged out. When you change a team member's permission from *full access* to *manage-only* or when you delete a team member's access, the team member can continue accessing Amplify Studio with their previously granted permissions until their token expires.
 </Callout>
+
+## Understanding how Studio manages user access
+
+<Callout warning>
+  The following resources are all managed by Studio so please **do not modify their settings, as they may affect your login experience**.
+</Callout>
+
+### User pool
+Studio manages user access using an Amazon Cognito User Pool in your account. You can invite up to 50,000 monthly users to Studio without cost.
+
+Studio manages user access using an Amazon Cognito User Pool in your account, named:
+- *amplify_backend_manager_APPID*.
+
+### IAM Roles
+In order to give the *Full access* and *Manage only* groups the necessary permissions, Studio creates 2 IAM roles, named:
+- *USERPOOLID_Full-access*
+- *USERPOOLID_Manage-only*
+
+### Cognito Identity Pool
+An Amazon Cognito Identity Pool is also created to vend AWS credentials that are tied to the *Full access* and *Manage only* groups, named:
+- *amplify_backend_manager_APPID*
+
+### Cognito Lambda triggers
+To provide a passwordless login experience from AWS Amplify Console to Amplify Studio, Studio creates 4 Cognito Lambda triggers *associated with the above-mentioned User Pool*, named:
+- *amplify-login-create-auth-challenge-SHORT_CODE*
+- *amplify-login-custom-message-SHORT_CODE*
+- *amplify-login-define-auth-challenge-SHORT_CODE*
+- *amplify-login-verify-auth-challenge-SHORT_CODE*
+
+### Recreating Studio managed resources
+If you happen to delete or otherwise mismanage any of the resources mentioned above, you can re-create them by first disabling and then re-enabling Studio for your Amplify Project on the Amplify management console.
```

Comment:
> If you happen to delete or otherwise mismanage any of the resources mentioned above

might not be appropriate to callout the customer here. We could maybe say `If your Studio application experiences any issues logging in or the resources have been deleted, you can re-create the resources by disabling and then re-enabling Studio for your Amplify Project on the Amplify management console.`